app/models/dialog/container_template_service_dialog.rb

```diff
@@ -0,0 +1,47 @@
+class Dialog
+  class ContainerTemplateServiceDialog
+    def self.create_dialog(label, params)
+      new.create_dialog(label, params)
+    end
+
+    # This dialog is to be used by a container template service
+    def create_dialog(label, params)
+      Dialog.new(:label => label, :buttons => "submit,cancel").tap do |dialog|
+        tab = dialog.dialog_tabs.build(:display => "edit", :label => "Basic Information", :position => 0)
+        add_parameters_group(tab, 0, params)
+        dialog.save!
+      end
+    end
+
+    private
+
+    def add_parameters_group(tab, position, params)
+      tab.dialog_groups.build(
+        :display  => "edit",
+        :label    => "Parameters",
+        :position => position
+      ).tap do |dialog_group|
+        params.each_with_index do |param, index|
+          add_parameter_field(param, dialog_group, index)
+        end
+      end
+    end
+
+    def add_parameter_field(param, group, position)
+      group.dialog_fields.build(
```

Comment:
Here are some parameter data.
```
#<ContainerTemplateParameter:0x007f8086adabe8
  id: 18,
  name: "MQ_USERNAME",
  ems_created_on: nil,
  display_name: nil,
  description: "User name for standard broker user. It is required for connecting to the broker. If left empty, it will be generated.",
  value: nil,
  generate: "expression",
  from: "user[a-zA-Z0-9]{3}",
  required: nil,
  container_template_id: 2,
  created_at: Mon, 08 May 2017 18:46:17 UTC +00:00,
  updated_at: Mon, 08 May 2017 18:46:17 UTC +00:00>,
 #<ContainerTemplateParameter:0x007f8086adaa08
  id: 19,
  name: "MQ_PASSWORD",
  ems_created_on: nil,
  display_name: nil,
  description: "Password for standard broker user. It is required for connecting to the broker. If left empty, it will be generated.",
  value: nil,
  generate: "expression",
  from: "[a-zA-Z0-9]{8}",
  required: nil,
  container_template_id: 2,
  created_at: Mon, 08 May 2017 18:46:17 UTC +00:00,
  updated_at: Mon, 08 May 2017 18:46:17 UTC +00:00>
```
After understanding how they should be used, I will add support for them. Maybe in a separate PR.